tests/acceptance/SubscriberCountShortcodeCest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Segment;
+use MailPoet\Test\DataFactories\Subscriber;
+
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../DataFactories/Subscriber.php';
+
+class SubscriberCountShortcodeCest {
+  function createSubscriberCountPageWithShortcode(\AcceptanceTester $I) {
+    $I->wantTo('Create page with MP subscriber shortcode');
+    $segment_factory = new Segment();
+    $segment = $segment_factory->withName('SubscriberCount')->create();
+    $subscriber_factory = new Subscriber();
+    $subscriber_factory->withSegments([$segment])->create();
+    $pageTitle='OurSubscribers';
+    $pageContent='[mailpoet_subscribers_count\ segments="' . $segment->id . '"]';
+    $I->cli('post create --allow-root --post_type=page --post_title=' . $pageTitle . '  --post_content=' . $pageContent);
+    $I->login();
+    $I->amOnPage('/wp-admin/edit.php?post_type=page');
+    $I->waitForText($pageTitle);
+    $I->click($pageTitle);
+    $I->click('Publish');
+    //see live page with shortcode output
+    $I->click('View page');
+    $I->waitForText($pageTitle);
+    $I->waitForText('1');
```

Comment:
I see what you did here, but I don't think this test is enough. Even if MP is disabled and the shortcode is not rendered the test would pass, because there is an Archive where I can see `1` in the year. 

I think we need to prepend some unique string in front of the shortcode and then assert the whole thing. Something like: `Mailpoet subscribers count 1`

Reply:
Excellent point, I'll fix.